grpc/execution/server_test.go

```diff
@@ -342,8 +342,143 @@ func TestExecutionServiceServerV1Alpha2_ExecuteBlock(t *testing.T) {
 	}
 }
 
+func TestExecutionServiceServerV1Alpha2_ExecuteBlockWithOutEnoughGas(t *testing.T) {
```

Comment:
This test instantiates with a very low gas limit, otherwise similar to the above